docs/exploring-gnuradio/dial_tone.py

```diff
@@ -41,5 +41,5 @@ def build_graph():
 if __name__ == '__main__':
     tb = build_graph()
     tb.start()
-    eval(input('Press Enter to quit: '))
+    input('Press Enter to quit: ')
```

Comment:
Unless, of course, if you `from builtins import input`. In that case though, you added 'future' as a dependency for GNU Radio on Py2k.